swatch-contracts/src/main/java/com/redhat/swatch/contract/product/UpstreamProductData.java

```diff
@@ -116,7 +119,10 @@ private UpstreamProductData(String sku) {
    */
   public static Optional<OfferingEntity> offeringFromUpstream(
       String sku, ProductDataSource productDataSource) {
-    log.debug("Retrieving product tree for offeringSku=\"{}\"", sku);
+    log.info(
+        "Received request on Message topic: {} to retrieving product tree for offeringSku=\"{}\"",
+        MDC.get(REQUEST_SOURCE),
```

Comment:
We don't need to do this, since the MDC keys will be automatically printed when exporting the traces to splunk.

Reply:
Updated.

Output: `INFO  [com.red.swa.con.ser.OfferingSyncService] (vert.x-worker-thread-1) {request_source=offering-sync-service-umb, sampled=true, spanId=31387913eb775341, traceId=ae2e834f391f39a44ac82c3db0153235} Received UMB message on offering-sync-service-umb for productSku=RH0180191INFO`